rpa-recorder/6.1/modules/ROOT/pages/user-interface.adoc

```diff
@@ -114,6 +114,37 @@ To continue recording, add a Bot Task.
 [NOTE]
 Pause the recording before adding an activity or changing the creation mode of the activities. If you open the pop-up using the respective hotkey, while the recording is running, the recording automatically pauses.
 
+== Web Recording Browser
+
+Use the Web Recording Browser to record web elements. It behaves like a normal browser and enables you to select additional elements of a website and determine their XPath.
```

Comment:
I'd suggest to use the word "regular" here, not "normal" and call it "web browser" (not just "browser"):
old: "... like a normal browser ..." >  new: "... like a regular web browser ..."